### PR TITLE
Add pcl::PointCloud to sensor_msgs::Image converter 

### PR DESCRIPTION
**Pull request by mikeferguson**
For extracting the rgb component of a cloud. This is a function I frequently use, that was previously provided by PCL (hence the header copyright adjustment).

### CODE REVIEW

**Comment by tfoote** [comment]
+1

File: include/pcl_conversions/pcl_conversions.h [hunk 1]
```diff
@@ -472,6 +473,34 @@ namespace pcl {
     pcl_conversions::moveFromPCL(pcl_image, image);
   }
 
+  template<typename T> void
+  toROSMsg (const pcl::PointCloud<T> &cloud, sensor_msgs::Image& msg)
+  {
+    // Ease the user's burden on specifying width/height for unorganized datasets
+    if (cloud.width == 0 && cloud.height == 0)
```
Comment (wjwwood) [inline include/pcl_conversions/pcl_conversions.h:480]: I know you are copying this from somewhere, but can you please add `{}` to this `if` statement. Mixing this with an `else` with `{}` really irks me.

**Comment by wjwwood** [comment]
Otherwise, +1

**Comment by mikeferguson** [comment]
@wjwwood I missed that one. I did also adjust the style before a little bit to better match the rest of pcl_conversions. Fixed and squashed back to a single commit.

**Comment by wjwwood** [comment]
Cool, looks ok to me.

**Comment by wjwwood** [comment]
Sorry, I probably should have waited for @paulbovbel or @LucidOne to review this first, since they are also helping to maintain.

If you guys see a problem with this, please let us know.

Otherwise, you guys should consider back and/or forward porting this to indigo and groovy, then figuring our when you'd like to release it.

**Comment by paulbovbel** [comment]
No problems on my end, thanks for keeping an eye out.

**Pull request merged**